resources/assets/less/status.less

```diff
@@ -0,0 +1,328 @@
+      width: 4px;
+      height: 74px;
+      top: 16px;
+      left: 16px;
+    }
+
+    &:after {
+      width: 4px;
+      height: 68px;
+      top: -68px;
+      left: 16px;
+    }
+  }
+
+  &__incident-content {
+    margin-top: 40px;
+    margin-bottom: 40px;
+
+    @media (max-width: @bigScreen) {
+      padding-left: 10px;
+    }
+  }
+
+  &__incident-info-date {
+    font-weight: bold;
+    font-style: italic;
+  }
+
+  &__incident-info-by {
+    font-weight: 500;
+    color: @username;
+    font-style: italic;
+  }
+
+  &__incident-desc {
+    font-size: 20px;
+    margin-left: 64px;
+
+    > span:first-child {
+      text-decoration: line-through;
+    
+      .status__incident-dest--active {
+        text-decoration: none;
+      }
+    }
+
+    > span:last-child {
+      text-transform: capitalize;
+    }
+
+    @media (max-width: @bigScreen) {
+      margin-left: 44px;          
+    }
+  }
+
+  &__map {
+    background: #333;
+  }
+
+  &__online {
+    > h1 {
+      padding-top: 20px;
+      padding-left: 20px;
+    }
+  }
+  
+  &__info {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+
+  &__info-border {
+    padding-bottom: 20px;
+    border-bottom: 2px solid @dark-pink;
+    width: 100%;
+    &:extend(.default-box-shadow);
+  }
+
+  &__info-separator {
+    width: 2px;
+    margin-top: 7px;
+    margin-bottom: 4px;
+    box-shadow: none;
+    background: linear-gradient(to bottom, rgba(255,255,255,0.47) 0%, rgba(169,169,169,0.47) 100%);
+  }
+
+  &__info-score, &__info-users {
+    flex: 1;
+    cursor: pointer;
+    text-align: center;
+    transition: all 250ms;
+    transition-timing-function: ease;
+    border-top: 14px solid transparent;
+
+    > h4 {
+      color: #333;
+      font-style: normal;
+    }
+
+    > h4, h1 {
+      font-weight: bold;
+    }
+
+    &--active {
+      border-top: 14px solid @dark-pink;
+    }
+  }
+
+  &__recent {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+
+  &__recent-incidents, &__recent-uptime {
+    width: 49.5%;
+    padding: 10px 20px;
+
+    @media (max-width: @bigScreen) {
+      width: 100%;
+    }
+  }
+
+  &__recent-uptime-when > *, &__recent-incidents-when > * {
+    display: inline;
+    margin-right: 22px;
+    color: #555;
+    font-size: 18px;
+    cursor: pointer;
+  }
+
+  &__recent-uptime--active, &__recent-incidents--active {
+    font-weight: bold;
+    color: @dark-pink;
+  }
+  
+  &__recent-incidents {
+    margin-right: 0.5%;
+
+    @media (max-width: @bigScreen) {
+      margin-right: 0;
+    }
+  }
+
+  &__recent-uptime {
+    margin-left: 0.5%;
+
+    @media (max-width: @bigScreen) {
+      margin-left: 0;
+    }
+  }
+
+  &__recent-charts {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+    
+    min-width: 350px;
+    min-height: 350px;
+    height: 100%;
+
+    > * {
+      width: 49%;

```

Comment:
just use `osu-layout__col` for this. Check `resources/assets/coffee/react/beatmaps/beatmap-listing.coffee` for usage.